src/rdb.c

```diff
@@ -755,7 +755,7 @@ ssize_t rdbSaveStreamPEL(rio *rdb, rax *pel, int nacks) {
     raxStart(&ri, pel);
     raxSeek(&ri, "^", NULL, 0);
     while (raxNext(&ri)) {
-        /* We store IDs in raw form as 128 big big endian numbers, like
+        /* We store IDs in raw form as 128 bit big endian numbers, like
```

Comment:
this was caught by the duplicate word forbidden pattern.

There's actually a second variation of this bug in https://github.com/valkey-io/valkey/blob/988297d19abea59799cdfaedcf7d0b01f0a559d8/src/t_stream.c#L1043

which I didn't see because the rule that complained here didn't see `" big big "` there.

I'll fix that instance. And probably _also_ add a `-` between `big` and `endian` here. (Plus probably apply that throughout the repository.)